src/libraries/System.Private.CoreLib/src/System/Collections/Concurrent/ConcurrentQueueSegment.cs

```diff
@@ -183,7 +186,7 @@ public bool TryDequeue([MaybeNullWhen(false)] out T item)
                 }
 
                 // Lost a race. Spin a bit, then try again.
-                spinner.SpinOnce(sleep1Threshold: -1);
+                spinner.SpinOnce(sleep1Threshold: _sleep1Threshold);
```

Comment:
With a very clearly named constant argument,  you can remove the "sleep1Threshold: " from each call site.